src/elements/content-picker/ContentPicker.js

```diff
@@ -95,6 +96,7 @@ type Props = {
     messages?: StringMap,
     onCancel: Function,
     onChoose: Function,
+    renderCustomActionButtons?: ({ selectedCount: number, selectedItems: BoxItem[] }) => Node,
```

Comment:
Could you also add this new prop to the readme for ContentPicker https://github.com/box/box-ui-elements/blob/master/src/elements/content-picker/README.md?